### DISCUSSION TITLE
problem loading channels

### DISCUSSION THREAD

**Discussion by RobbyThomas2525** [iBicha/playlet#490] [Q&A] [answered]
the problem i'm having is loading channels on playlet videos are still working but the channels will not let me load into them it doesn't matter what server setting i have it on or anything it still does it

**Comment by iBicha** [comment, marked as answer]
Can you share which server did you try? If you're using a public one from https://api.invidious.io/ it must be updated after Nov 10th, because that's when the fix was merged (fix: https://github.com/iv-org/invidious/pull/5059)
If this is not the issue, please share more info, such as your invidious and error messages, I'll look into it
